files/hipchat.rb

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+
+require "#{File.dirname(__FILE__)}/base"
+#require_relative 'base'
```

Comment:
it's wishful thinking of a day when we don't support ruby 1.8.7 :D 

will remove it.